### PR TITLE
Add org.filmulator.Filmulator

### PR DESCRIPTION
**Pull request by hfiguiere**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributible by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/CarVac/filmulator-gui/issues/126
- [x] ~~I own~~ the domain used in the application ID _is upstream's_ or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build io.github.CarVac.Filmulator

**Comment by hfiguiere** [comment]
bot, build org.filmulator.Filmulator

**Comment by hfiguiere** [comment]
bot, build org.filmulator.Filmulator

**Comment by hfiguiere** [comment]
The appstream file is submitted here.

There other patches might become obsolete when upstream move to CMake.

**Comment by hfiguiere** [comment]
aarch64 build is disabled for now because of this upstream issue:

https://github.com/CarVac/filmulator-gui/issues/128

**Comment by hfiguiere** [comment]
bot, build org.filmulator.Filmulator

File: org.filmulator.Filmulator.json [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    "app-id": "org.filmulator.Filmulator",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "filmulator-gui",
+    "rename-desktop-file": "filmulator-gui.desktop",
+    "rename-icon": "filmulator64icon",
+    "finish-args": [
+        /* X11 + XShm access */
+        "--share=ipc",
```
Comment (nedrichards) [inline org.filmulator.Filmulator.json:11]: you don't actually need that share when doing fallback-x11. it's included with that permission (separating it in x11 was a mistake but... backwards compatibility!).

Comment (hfiguiere) [reply org.filmulator.Filmulator.json:11]: good to know.

**Comment by hfiguiere** [comment]
Tested on RPi 400 as well

**Comment by hfiguiere** [comment]
bot, build org.filmulator.Filmulator

**Comment by hfiguiere** [comment]
I believe this is ready for review.

**Comment by hfiguiere** [comment]
bot, build org.filmulator.Filmulator

**Pull request closed without merging**